package.json

```diff
@@ -23,7 +23,7 @@
   "license": "NASA-1.3",
   "repository": "nasa-gibs/worldview",
   "engines": {
-    "node": ">=10.16.3"
+    "node": ">=12.21.0 <=14.16.0"
```

Comment:
Ok so it looks like we have our own script that is using this to check for the Node version and it throws a warning.  I think we should probably remove this since this capability is baked into NPM now (maybe it wasn't 4 years ago when we added that script).  

I configured the range to indicate which versions we are testing with CI (12 and 14) since that's what we'll be regularly checking for compatibility.  If a developer is using a version greater than 14.16.0 they'll see something like:

```
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'worldview@3.9.0',
npm WARN EBADENGINE   required: { node: '>=12.21.0 <=14.16.0' },
npm WARN EBADENGINE   current: { node: 'v15.11.0', npm: '7.6.0' }
npm WARN EBADENGINE }
```

... otherwise they shouldn't see any warning message.  Though for some reason I'm not getting the warning when I use a version _earlier_ than what's defined in `engines`. I will look at that some more.

Thoughts?

Reply:
I can't seem to trigger the `EBADENGINE` warning no matter how I set the range.

Reply:
Ok after some more investigation it looks like those warnings only get triggered if you are using Node 15+.  I am going to update our `chekNodeVersion` script to handle the range syntax and it will still throw a warning if you are below, but will rely on Node to show the warning if you are using 15+.  Here's some example output of this in action:
```
➜  worldview git:(2689-github-actions-squashed) ✗ node --version
v10.24.0
➜  worldview git:(2689-github-actions-squashed) ✗ node tasks/checkNodeVersion.js
 
WARN  The suggested version of node for the installation of Worldview is   >=12.21.0 <=14.16.0   you are using   v10.24.0
 If you have difficulties installing Worldview, please try using the install again using a compatible node version   >=12.21.0 <=14.16.0 
➜  worldview git:(2689-github-actions-squashed) ✗ nvm use 12
Now using node v12.21.0 (npm v6.14.11)
➜  worldview git:(2689-github-actions-squashed) ✗ node tasks/checkNodeVersion.js
Preinstall conditions satisfied. Installing... 
➜  worldview git:(2689-github-actions-squashed) ✗ nvm use 14
Now using node v14.16.0 (npm v6.14.11)
➜  worldview git:(2689-github-actions-squashed) ✗ node tasks/checkNodeVersion.js
Preinstall conditions satisfied. Installing... 
➜  worldview git:(2689-github-actions-squashed) ✗ nvm use 15
Now using node v15.11.0 (npm v7.6.0)
➜  worldview git:(2689-github-actions-squashed) ✗ node tasks/checkNodeVersion.js
Preinstall conditions satisfied. Installing... 
➜  worldview git:(2689-github-actions-squashed) ✗ npm ci
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'worldview@3.9.0',
npm WARN EBADENGINE   required: { node: '>=12.21.0 <=14.16.0' },
npm WARN EBADENGINE   current: { node: 'v15.11.0', npm: '7.6.0' }
npm WARN EBADENGINE }
```

Reply:
Great, that change looks to handle it well.